ink-wrapper-types/Cargo.toml

```diff
@@ -14,14 +14,19 @@ categories = []
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-async-trait = "0.1.64"
-ink_primitives = "4.2.1"
-scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
-pallet-contracts-primitives = { version = "7.0.0" }
-aleph_client = { version = "3.0.0", optional = true }
+aleph_client = { git = "https://github.com/Cardinal-Cryptography/aleph-node", package = "aleph_client", tag = "r-12.1", optional = true }
+
 anyhow = { version = "1.0.51", optional = true }
-subxt = { version = "0.25.0", optional = true }
+async-trait = "0.1.64"
+
+ink_primitives = "4.3.0"
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = [
+    "derive",
+] }
+pallet-contracts-primitives = { git = "https://github.com/Cardinal-Cryptography/substrate.git", package = "pallet-contracts-primitives", branch = "aleph-v1.0.0", default-features = false, features = [
+    "std",
+] }
+subxt = { version = "0.30.1", optional = true }
 
 [features]
-default = ["aleph_client"]
```

Comment:
This might be unnecessary but I find it cleaner now that I can choose with which feature the `ink-wrapper-types` are built. Adding `drink!` would mean adding a new feature here as well.